### PR TITLE
Add docs and settings.

### PR DESCRIPTION
**Pull request by wRAR**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -12,3 +12,57 @@ Requirements
 * Python 3.8+
 * Scrapy 2.11+
 * zyte-spider-templates
+
+You also need a `Zyte API`_ subscription.
```
Comment (wRAR) [inline README.rst]: I'm not sure if you can use this repo without Zyte API?

File: README.rst [hunk 2]
```diff
@@ -12,3 +12,57 @@ Requirements
...
+
+.. _Zyte API: https://docs.zyte.com/zyte-api/get-started.html
+
+Usage
+=====
+
+.. note:: As some modifications to the files in this repo are needed before it
+   can be used, it's recommended to fork it.
```
Comment (wRAR) [inline README.rst]: Should this explain what forking is?

File: README.rst [hunk 3]
```diff
@@ -12,3 +12,57 @@ Requirements
...
+For more information and more verbose descriptions of specific steps you can
+check:
+
+* `The Scrapy Cloud tutorial
+  <https://docs.zyte.com/web-scraping/tutorial/cloud.html>`_.
+* `The shub documentation <https://shub.readthedocs.io/>`_.
+* `The Zyte API documentation
+  <https://docs.zyte.com/zyte-api/get-started.html>`_.
+* `The zyte-spider-templates documentation
+  <https://github.com/zytedata/zyte-spider-templates>`_.
```
Comment (wRAR) [inline README.rst:86]: Any other links here?

File: zyte_crawlers_project/settings.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
+# Scrapy settings for zyte_crawlers_project project
```
Comment (wRAR) [inline zyte_crawlers_project/settings.py]: This currently follows the default Scrapy template, as opposed to https://github.com/zytedata/zyte-crawlers/blob/main/zyte_crawlers/settings.py

File: README.rst [hunk 5]
```diff
@@ -2,13 +2,91 @@
...
+#.  Make sure you have a Zyte API subscription. For Scrapy Cloud runs the API
+    key will be used automatically, for local runs you need to set a setting or
+    an environment variable, as described in the first steps above.
+#.  Run ``shub login`` and enter your `Scrapy Cloud API key
+    <https://app.zyte.com/o/settings/apikey>`_.
+#.  Deploy your project with ``shub deploy 000000``, replacing ``000000`` with
+    your Scrapy Cloud project ID (found in the project dashboard URL).
+    Alternatively, put the project ID into the ``scrapinghub.yml`` file to be
+    able to run simply ``shub deploy``.
+#.  Now you should be able to create smart spiders on your Scrapy Cloud project
```
Comment (wRAR) [inline README.rst:73]: This should link to some docs on this after we publish them.

File: README.rst (hunk 1 above)
Comment (Gallaecio) [reply README.rst]: Technically yes, so we just need to be careful with the wording, e.g.

```suggestion
You also need a `Zyte API`_ subscription for Zyte API features, including AI-powered spiders.
```

File: zyte_crawlers_project/settings.py [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    # spiders defined in this project
+    "zyte_crawlers_project.spiders",
+]
+NEWSPIDER_MODULE = "zyte_crawlers_project.spiders"
+
+# Crawl responsibly by identifying yourself (and your website) on the user-agent
+#USER_AGENT = "zyte_crawlers_project (+http://www.yourdomain.com)"
+
+# Obey robots.txt rules
+ROBOTSTXT_OBEY = True
```
Comment (Gallaecio) [inline zyte_crawlers_project/settings.py]: This is problematic. In fact, I’m considering mentioning it on the tutorial, as it can bite users in the ass.

Comment (wRAR) [reply zyte_crawlers_project/settings.py]: What are the potential problems?

Comment (Gallaecio) [reply zyte_crawlers_project/settings.py]: The spider may refuse to work for a given website, or for specific pages of it, based on the robots.txt spec. And it is easy, specially for starting Scrapy users, to miss the warning about it in the logs.

File: scrapinghub.yml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# Configure your Zyte Cloud projects here,
+# see https://shub.readthedocs.io/en/stable/configuration.html
+# and https://docs.zyte.com/web-scraping/tutorial/cloud.html
+
+#projects:
+#  default: NNNNNN
+stacks:
+  default: scrapy:2.11
```
Comment (kmike) [inline scrapinghub.yml]: Should we use the pinned version of the stack for now?

File: zyte_spider_templates_project/settings.py [hunk 8]
```diff
@@ -0,0 +1,119 @@
+# Scrapy settings for zyte_spider_templates_project project
```
Comment (kmike) [inline zyte_spider_templates_project/settings.py]: My preference would be to go minimal, like in #2.

File: README.rst [hunk 9]
```diff
@@ -2,13 +2,109 @@
...
+    ``scrapy.cfg`` and ``<project ID>/settings.py`` (``SCRAPY_POET_DISCOVER``
+    and ``SPIDER_MODULES`` settings) accordingly.
+
+#.  For local development, assign your `Zyte API key
+    <https://app.zyte.com/o/zyte-api/api-access>`_ to the ``ZYTE_API_KEY``
+    environment variable, for example, using `direnv <https://direnv.net/>`_.
+
+    .. note:: `Scrapy Cloud
+        <https://docs.zyte.com/scrapy-cloud/get-started.html>`_
+        automatically configures your API key through a Scrapy setting.
```
Comment (kmike) [inline README.rst]: It also does it through env variable.

```suggestion
        automatically provides Zyte API key for the jobs, if you have a subscription.
```

File: README.rst (hunk 3 above)
Comment (kmike) [reply README.rst:86]: I think we should also link to Scrapy docs.

**Review by kmike — APPROVED** [review]

**Pull request merged**